docs/src/pages/guides/cli-options.md

```diff
@@ -72,7 +72,7 @@ You use the `-H` or `--har` to specify a HAR file.
 monika -H my-file.har
 ```
 
-You can use the combination of --create-config and --har flags to convert the HAR archive into to a monika.json configuration file.
+You can use the combination of --create-config and --har flags to convert the HAR archive into to a monika.yml configuration file.
```

Comment:
again, is it yaml or yml? need to be consistent all over the place.

Reply:
done.. made to be yml.